lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -359,6 +360,9 @@ def write(chunk)
             if record.key?('kubernetes')
               handle_container_metadata(record, entry)
             end
+            # Save the timestamp if available
+            timestamp = nil
+            timestamp = record['time'] if record.key?('time')
```

Comment:
@bokowski Not going to hold up the merge because of this, but wondering why you didn't use the ternary operator or an if expression here...